DashSync/shared/Models/Wallet/DSAccount.m

```diff
@@ -1004,15 +1004,26 @@ - (DSTransaction *)proposalCollateralTransactionWithData:(NSData *)data {
 
 // returns an unsigned transaction that sends the specified amounts from the wallet to the specified output scripts
 - (DSTransaction *)updateTransaction:(DSTransaction *)transaction forAmounts:(NSArray *)amounts toOutputScripts:(NSArray *)scripts withFee:(BOOL)fee {
-    return [self updateTransaction:transaction forAmounts:amounts toOutputScripts:scripts withFee:fee shuffleOutputOrder:YES];
+    return [self updateTransaction:transaction forAmounts:amounts toOutputScripts:scripts withFee:fee shuffleOutputOrder:YES followBIP69sorting:YES];
 }
 
-- (DSTransaction *)updateTransaction:(DSTransaction *)transaction forAmounts:(NSArray *)amounts toOutputScripts:(NSArray *)scripts withFee:(BOOL)fee shuffleOutputOrder:(BOOL)shuffleOutputOrder {
-    return [self updateTransaction:transaction forAmounts:amounts toOutputScripts:scripts withFee:fee toShapeshiftAddress:nil shuffleOutputOrder:shuffleOutputOrder];
+- (DSTransaction *)updateTransaction:(DSTransaction *)transaction
+                          forAmounts:(NSArray *)amounts
+                     toOutputScripts:(NSArray *)scripts
+                             withFee:(BOOL)fee
+                  shuffleOutputOrder:(BOOL)shuffleOutputOrder
+                  followBIP69sorting:(BOOL)followBIP69sorting {
```

Comment:
Maybe an enum here would be cleaner? Because you can really only have one of 3 choices right now.